Sage/carbon
# fix(popover-container): popover position re-renders with controlled MultiSelect

fix #6793

### Proposed behaviour



- Selecting an option in a  controlled `MultiSelect` within a Popover container does not cause an unintended animation. 

### Current behaviour



- When a controlled `MultiSelect` is placed inside a `PopoverContainer` and an option is selected, the popover re-renders with an unintended animation.

### Checklist




#### QA


### Additional context



### Testing instructions



Cannot be tested with RTL/Playwright so manual testing needed - see 'with multi select' test story.
